script.js

```diff
@@ -1,3 +1,43 @@
+var minWidth = 250;
+
+function alignBorders(identifier, className) {
+  const currentBorder = document.getElementById(identifier);
+  const currentColumn = document.getElementsByClassName(className);
+  const rect = currentColumn[0].getBoundingClientRect();
+  if (identifier == 'columnline-five') {
+    const currentColumnOffsetLeft = rect.left + rect.width;
+    currentBorder.style.left = currentColumnOffsetLeft + 'px';
+
+    return;
+  }
+
+  const currentColumnOffsetLeft = rect.left;
+  currentBorder.style.left = currentColumnOffsetLeft + 'px';
+}
+
+function addLoadEvent(func) {
+  var oldonload = window.onload;
+  if (typeof window.onload != 'function') {
+    window.onload = func;
+  }
+  else {
+    window.onload = function() {
+      if (oldonload) {
+        oldonload();
+      }
+      func();
+    }
+  }
```

Comment:
I think you can replace this with a simple addEventListener for onload on window. But I am not sure.